components/base/kinematicbase/ptgKinematics.go

```diff
@@ -203,6 +203,9 @@ func (ptgk *ptgBaseKinematics) goToInputs(ctx context.Context, inputs []referenc
 	}
 	arcSteps := ptgk.trajectoryToArcSteps(selectedTraj)
 
+	// Loop label is required because `break` in the inner `for` loop will not
+	// be able to break out of the outer `for` loop.
+trajectory:
```

Comment:
I am not entirely sure if we should have this but I wanted to call it out.
Since we not longer have  `PanicCapturingGo` we would end up trying to send a `SetVelocity` call to a base with a cancelled context.